descriptors/ModuleDescriptor-template.json

```
@@ -106,6 +106,24 @@
         }
       ]
     },
+    {
+      "id": "change-due-date",
```

This interface appears to be more specifically about changing the due date of multiple loans in a batch. I suggest that be more explicitly reflected in the name of the interface, e.g.
```suggestion
      "id": "batch-change-loan-due-dates",
```

---

Renamed.